lang/python/python-requests/patches/0001-idna-dependency-bump.patch

```diff
@@ -12,43 +12,15 @@ https://github.com/psf/requests/pull/5711
  setup.py | 2 +-
  1 file changed, 1 insertion(+), 1 deletion(-)
 
-diff --git a/setup.py b/setup.py
-index 7ba4b2a25f..f265384236 100755
 --- a/setup.py
 +++ b/setup.py
-@@ -43,7 +43,7 @@ def run_tests(self):
-
+@@ -43,7 +43,8 @@ packages = ['requests']
+ 
  requires = [
      'chardet>=3.0.2,<5',
 -    'idna>=2.5,<3',
-+    'idna>=2.5,<4',
-     'urllib3>=1.21.1,<1.27',
-     'certifi>=2017.4.17'
-
-
-From d3e00a4958af046879f24de365d5589d861ea6ef Mon Sep 17 00:00:00 2001
```

Comment:
This needs to be split to two patches. Notice how quilt is merging the patches here.